week3/intro_to_costing.ipynb

```diff
@@ -0,0 +1,779 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "f4e4a087-5dcd-45d1-924e-59f0302172b5",
+   "metadata": {},
+   "source": [
+    "# Introduction to WaterTAP Costing"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "edf7f369-cdfc-4c02-b65f-3e2184629f33",
+   "metadata": {},
+   "source": [
+    "# DELETE THIS CELL BEFORE MERGING\n",
+    "Outline:\n",
+    "- Brief summary of WaterTAP costing - slides will hopefully cover most of the details\n",
+    "- Just build an RO flowsheet\n",
+    "- Walk through adding the existing costing for each unit model (consider showing snippets of the unit model costing files or documentation with our explanations)\n",
+    "- Walk through how to modify the existing parameters\n",
+    "- How to add system level metrics (LCOW, SEC, etc.)\n",
+    "- Walk through how to go from simulating costing to optimizing it (add objective function for minimizing LCOW)\n",
+    "- Add a try it yourself for optimizing for a different objective (SEC)\n",
+    "- Walk through how they might add their own costing (register their own flow types) as shown in #2 here: https://watertap.readthedocs.io/en/latest/technical_reference/costing/costing_base.html\n",
+    "\n",
+    "## TODO: Mutka should look into adding capital cost components dynamically\n"
```

Comment:
This is a brainstorming outline that Mukta and I created. It should be deleted or refined before merging this PR